### DISCUSSION TITLE
Duplicate offer item

### DISCUSSION THREAD

**Discussion by johnnywo** [lovata/oc-shopaholic-plugin#349] [Ideas]
Hi, it would be really nice if I just could clone an existing offer item (and apply small modifications afterwards). Instead of running the process of creating a new one each time. 

Thank you,
Emil

**Comment by lautsevich** [comment]
@johnnywo, such a good idea. We'll develop it some. Feel free to develop it by yourself and send a PR.

**Comment by johnnywo** [comment]
@lautsevich thank you - if I feel able to do that I will!
